editor/plugins/animation_blend_space_2d_editor.cpp

```diff
@@ -820,9 +820,7 @@ void AnimationNodeBlendSpace2DEditor::_notification(int p_what) {
 
 			String error;
 
-			if (!tree) {
-				error = TTR("BlendSpace2D does not belong to an AnimationTree node.");
```

Comment:
If it would be better to remove the other check, let me know, then I will adjust the PR.
Initially I didn't want to change change current behavior with this PR.

Reply:
CC @TokageItLab

Reply:
The `!tree` check is also present in several other AnimationNodeEditors.

They are not always false. But what this PR is trying to remove is simply the dead code part.

The explanation may have confused the terminology. This should not change the behavior, and is an acceptable change.